pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: flutter_ai_toolkit
 description: >-
   A set of AI chat-related widgets for Flutter apps targeting
```

Comment:
"A set of" seems a bit superfluous, perhaps just "A set of AI chat-related widgets for Flutter apps targeting" ?

Reply:
updated. ptal.